infra/modular-examples/gcp-google-workspace/main.tf

```diff
@@ -250,6 +251,95 @@ module "worklytics-psoxy-connection-long-auth" {
 }
 # END LONG ACCESS AUTH CONNECTORS
 
+# BEGIN MSFT Connectors
```

Comment:
This can be dropped from this example. In practice, we don't have any one who wants to host in GCP and consume data from MSFT. Hence there's `aws` examples for `msft` and `google`; but only a `gcp` example for `google`.

YAGNI. Let's leave it out until we have a need, rather than maintain it.

The MSFT case we do have is a `infra/examples/msft-365`, which is example to create MSFT app registrations alone - without all the corresponding AWS infra. This is to support the case where the person managing the AWS infra lacks permissions to do the MSFT Azure AD stuff. That's the other example we may want to modify to support this auth scheme - but I think leave it for later PR.